### ISSUE TITLE
Too many files are being moved to the Sonarr Recycle-bin for an upgrade, leaving no episode behind in the folder

### ISSUE THREAD

**Issue by TheGeneral007** [Sonarr/Sonarr#5004]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

C1. I cannot figure out when the logfiles tell me that the quality is WebRip-1080p and this comes from parsing the name of the file and it parsed that properly, what else can cause all the files to be a candidate for an upgrade. This could be the case for the S04E17 because there was a HDTV-1080p in the folder, but no matter what I try it keeps moving all the episodes to the RecycleBin-folder.
I start to think that because S04E17 is upgradable the program doesn't distinguish and deletes them all.

### Expected Behavior

E1. When I unmonitor episodes, I expect them to be safe from interference and not start deleting anything there.
E2. When I monitor episodes, I want the highest quality possible according to the Quality Profile Table 

### Steps To Reproduce

New Amsterdam (2018) - S04E01~S04E16  are unmonitored; S04E17 and up are monitored. 
I do a Rescan and then I put the files S04E01~S04E16 from the RecycleBin back to the folder:
\\synasmg21\video\Recyclebin\New Amsterdam (2018)\Season 04\
and immediately the files are being moved from \\synasmg21\video\Series to \\synasmg21\video\RecycleBin

### Environment

```markdown
- OS: Windows 10
- Sonarr: v3.0.8.1507
- Docker Install: No
- Using Reverse Proxy: No
- Browser: Firefox
```

### What branch are you running?

Main

### Trace Logs?

Trace Logs:
[20220428_1918](https://pastebin.ubuntu.com/p/qcfQQzwQy4/)
[20220428_1919](https://pastebin.ubuntu.com/p/CJ34JqfgWZ/)
[20220428_1922](https://pastebin.ubuntu.com/p/9J92XMZmn9/)
[20220428_1923](https://pastebin.ubuntu.com/p/S3XfzJqQ7c/)

### Anything else?

![image](https://user-images.githubusercontent.com/82135186/165808385-48a53e12-ca4a-48d9-8726-035c100eb24e.png)
![image](https://user-images.githubusercontent.com/82135186/165808558-4b414dc5-2371-4089-9a14-a35dc0aba56e.png)
![image](https://user-images.githubusercontent.com/82135186/165808643-eff06189-eea9-4e79-b1c4-6b1453631017.png)
![image](https://user-images.githubusercontent.com/82135186/165809367-27064fd5-627e-4e0b-8a64-ae8c1df57a5d.png)
![image](https://user-images.githubusercontent.com/82135186/165811040-8b9603ab-51be-4bee-b731-89fd135ad0de.png)
![image](https://user-images.githubusercontent.com/82135186/165814767-05f51ab2-92f7-4993-9f92-3ee2f0d3e0a2.png)

**Comment by markus101** [comment]
Please use one of the support channels: [forums](https://forums.sonarr.tv/), [subreddit](https://www.reddit.com/r/sonarr/), [discord ](https://discord.gg/Ex7FmFK), or [IRC](https://web.libera.chat/?channels=#sonarr) for support/questions.

This is an issue with your configuration, SAB is sorting files to your series folder and then Sonarr is trying to import from there.

> E1. When I unmonitor episodes, I expect them to be safe from interference and not start deleting anything there.

Unmonitoring tells Sonarr to not grab releases for that episode, it does not prevent it from importing an upgrade (or sidegrade).

> E2. When I monitor episodes, I want the highest quality possible according to the Quality Profile Table

That is how it works...

TLDR, fix your SAB config and _disable_ TV sorting. Do not have SAB download or sort files into the sorted series folders that Sonarr is managing.
